File: app/src/main/java/com/droidapp/MyApp.java
package com.droidapp;

import android.app.Application;

import com.droidnet.old.DroidNet;

/**
 * Created by aa on 29/04/17.
 */

public class MyApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        DroidNet.init(this);
    }

    @Override
    public void onLowMemory() {
        super.onLowMemory();
        DroidNet.getInstance().removeAllInternetConnectivityChangeListeners();
    }
}

File: app/src/main/java/com/droidapp/MyApp.kt
package com.droidapp

import android.app.Application
import com.droidnet.new.NetworkStateHolder.DroidNetConnectivityBroadcaster

/**
 * Created by aa on 29/04/17.
 */
class MyApp : Application() {
    override fun onCreate() {
        super.onCreate()

        //register DroidNet observer
        DroidNetConnectivityBroadcaster()
    }

}